utils/tim_delay.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * tim_delay.c
+ *
+ *  Created on: 19-Feb-2021
+ *      Author: Sonal Shrivastava
+ */
+
+#include <stdint.h>
+
+#include "drivers/mss_timer/mss_timer.h"
+#include "CMSIS/system_m2sxxx.h"
+#include "tim_delay.h"
+#include "led.h"
```

Comment:
Can be removed once `delay_test()` is removed. I'm assuming here the plan is to replace the use of MSS_TIM2 in `led` with calls to `delay()`, hopefully I'm right...

Reply:
Yup, you are absolutely right about that :)

Reply:
This line is still present in the latest version of the code. Better to remove it to avoid later confusion.

Reply:
Yes, I will remove it.